CODE_OF_CONDUCT.md

```diff
@@ -1,13 +1,51 @@
-# Code of Conduct
+# Overview
 
-> Online version of this document is available at https://sensu.io/conduct
+As a member of the Sensu community, we ask that you keep the following in mind:
+
+  * Be kind and welcoming to others.
+  * Respect personal space and ask before you enter it. Remember not
+    everyone likes to be touched or hugged or joked with.
+  * Be mindful of language. Do not disparage others. Listen more than you talk.
+  * Remember that harassment and sexist, racist comments or behavior are not
+    appropriate in our Community.
+
+Sensu in-person and online events are intended for networking and
+collaboration in the monitoring community. Attendees violating these rules may
+be asked to leave any event we host without a refund at the sole discretion of
+the event organizers.
+
+The same rules apply when participating in other Sensu-related forums and
+mailing lists, contributing to code and documentation, and in private
+correspondence.
```

Comment:
To be clear, that part is "private correspondence" is limited to Sensu related space ? (asking due to the Drupal 2017 controversy). 

Like, if I send you a email that would be a CoC violation, would it be covered by the CoC ?